src/SampleViewExtension/SampleWindowViewModel.cs

```diff
@@ -19,6 +19,16 @@ public SampleWindowViewModel(ReadyParams p)
             p.CurrentWorkspaceModel.NodeRemoved += CurrentWorkspaceModel_NodesChanged;
         }
 
+        public int getNodeCount()
```

Comment:
why do this? As far as I can tell you are just reimplementing count.

Reply:
I don't think count is available on `IEnumerable`

Reply:
then do - ToList() and then call count.

Reply:
You can use a helper function if you need to do additional work for each node but to just get the count `Nodes.ToList().Count()`

Reply:
ok. i will make the adjustments...give me a bit